.github/workflows/e2e_ios.yml

```diff
@@ -36,11 +36,17 @@ jobs:
   test:
     needs: checksecret
     if: needs.checksecret.outputs.is_SECRETS_PRESENT_set == 'true'
-    runs-on: macos-latest
-    # Kill the task if not finished after 60 minutes
-    timeout-minutes: 60
+    runs-on: macos-13
+    # Kill the task if not finished after 150 minutes
+    timeout-minutes: 150
```

Comment:
I have never seen any of our e2e tests take this long, and while the e2e-iOS tests for this branch seem about twice as slow as main, they don't seem to be exceeding 40 minutes, so why is this necessary?

Reply:
I did see some runs where the e2e failed with error message  "timeout exceeded". So I kept ramping up until I could see for most failed runs an error message that had nothing to do with timeouts. For a successful run this number is indeed way to high. I don't know what is a good number here, or at what point we think a timeout means failure.